### DISCUSSION TITLE
Flightplan with Backtrace in A32nx v0.8.1.

### DISCUSSION THREAD

**Discussion by HowlingWolf136** [flybywiresim/aircraft#7264] [Help]
Hi, I create Simbrief Flightplans and import them ito the mcdu on A32nx newest version (see above) and A339 from Headwind (also newest.
The importet Flightplan is the not correct as I have backtraces and other strange thing in it.
Please check screenshots
![Screenshot 2022-05-31 131356](https://user-images.githubusercontent.com/101331521/171176544-c27acbf8-fa54-4622-8d10-92da0782a71f.jpg)
![Screenshot 2022-05-31 130911](https://user-images.githubusercontent.com/101331521/171176561-63994c54-2692-4fe9-afb9-02b858a17f60.jpg)

**Comment by tracernz** [comment]
Please post the route from simbrief, including procedures, and screenshots of each page of the MCDU F-PLN. The second one is purely a graphical glitch (will fly fine)... the first looks like an out-of-sequence waypoint (BOA).

**Reply by HowlingWolf136** [reply]
Allright, will redo. Might Take an hour or so.

**Comment by HowlingWolf136** [comment]
This is the Simbrief Route 

![Screenshot 2022-06-01 124700](https://user-images.githubusercontent.com/101331521/171387278-d867f057-4a4e-42cc-9cb9-9b01a1b32633.jpg)

**Comment by HowlingWolf136** [comment]
This Time TRA is the Problem,
![Screenshot 2022-06-01 125505](https://user-images.githubusercontent.com/101331521/171388684-cbe8f56e-9a67-4f7d-8934-9b01a40e70ef.jpg)
![Screenshot 2022-06-01 125349](https://user-images.githubusercontent.com/101331521/171388701-a24ed7b9-127e-4d9e-b7fb-24a132fb6f91.jpg)
![Screenshot 2022-06-01 125319](https://user-images.githubusercontent.com/101331521/171388720-fe95254d-6341-4874-b5fb-b6a3e10ee140.jpg)
![Screenshot 2022-06-01 125252](https://user-images.githubusercontent.com/101331521/171388729-afbaf24c-c056-42d3-8de9-406c679241db.jpg)
![Screenshot 2022-06-01 125221](https://user-images.githubusercontent.com/101331521/171388742-cba515cd-c724-4bcb-9cf1-cccc84f637de.jpg)
![Screenshot 2022-06-01 125152](https://user-images.githubusercontent.com/101331521/171388760-55f19eb4-04c5-45e0-844a-0fb441528d2b.jpg)
![Screenshot 2022-06-01 124700](https://user-images.githubusercontent.com/101331521/171388775-6bb1fd3e-8c24-4a2b-8f8c-7a425bba61e2.jpg)
![Screenshot 2022-05-31 131356](https://user-images.githubusercontent.com/101331521/171388789-5c24ecc7-aca4-4d3a-a050-2c24c606397e.jpg)

**Comment by HowlingWolf136** [comment]
Here again (devolpment Version) Between Rosas, BOA and SAFOR (Safor Approach came later and got included thru the Approach selection in the MCDU


![Screenshot 2022-06-01 145755](https://user-images.githubusercontent.com/101331521/171410742-34ba28f5-f262-495e-a42f-d88c584cbe09.jpg)
![Screenshot 2022-06-01 145826](https://user-images.githubusercontent.com/101331521/171410754-bb00f140-64b0-4a3a-af84-dbea04dd43f6.jpg)
![Screenshot 2022-06-01 145853](https://user-images.githubusercontent.com/101331521/171410766-fa924b45-9783-436e-b160-dbc92bfa426d.jpg)
![Screenshot 2022-06-01 145915](https://user-images.githubusercontent.com/101331521/171410782-a281dfb0-f269-4f60-b6f8-2758f4e2e367.jpg)
![Screenshot 2022-06-01 145941](https://user-images.githubusercontent.com/101331521/171410799-90374c6c-d73c-4ee3-a3cd-b832233f82e2.jpg)
![Screenshot 2022-06-01 150009](https://user-images.githubusercontent.com/101331521/171410812-79daf876-659c-4c7d-8910-956ccdd8435c.jpg)
![Screenshot 2022-06-01 150034](https://user-images.githubusercontent.com/101331521/171410827-6e0aae4b-4bd3-4e60-a1ac-6f16f3d40ce3.jpg)
![Screenshot 2022-06-01 150057](https://user-images.githubusercontent.com/101331521/171410844-bfb1d71f-7477-4321-a421-4e3f134c56fa.jpg)
